test/jdk/java/lang/Thread/virtual/ThreadPollOnYield.java

```diff
@@ -62,6 +64,9 @@ void testThreadYieldPolls() throws Exception {
         done.set(true);
         vthread.join();
 
+        if (flag != true) {
+            throw new RuntimeException("flag = " + flag);
+        }
```

Comment:
What are these other changes about?

Reply:
Reason description as https://github.com/openjdk/jdk/pull/23550#issuecomment-2650141187.

Reply:
Can you remove this change from the PR as it this is a separate discussion? My guess is that in your environment the Thread.yield is somehow taking more than 5 seconds, is that right? In that case, we can modify the test to use a latch to ensure that it loops at least once. Anyway, separate issue, we shouldn't include it in this PR.

Reply:
Okey, the additional check has been removed.

Reply:
I have created a separate [issue](https://bugs.openjdk.org/browse/JDK-8349787) to record that. I will investigate later.